swarm.sh

```diff
@@ -50,6 +50,11 @@ function deploy_importers() {
   for service_path in "${COMPOSE_FILE_PATH}/importer/"*/; do
     target_service_name=$(basename "$service_path")
 
+    if [[ $IMPORT_ONLY != "" && "${target_service_name}" != $IMPORT_ONLY ]]; then
```

Comment:
Has the IMPORT_ONLY variable be declared?

Reply:
Yes it is declared in the .env.* files